BPR.Mediator/Models/ViolationModel.cs

```diff
@@ -2,6 +2,11 @@
 
 namespace BPR.Mediator.Models;
 
-public record ViolationModel(ViolationType Type, string Description, ViolationSeverity Severity, string Code, string File)
+public class ViolationModel
 {
+    public ViolationType Type { get; set; }
+    public string Description { get; set; } = default!;
+    public ViolationSeverity Severity { get; set; }
+    public string Code { get; set; }
```

Comment:
I guess it would be a good idea to set the string values so that they don't throw any warnings